lib/api/interceptor.ts

```diff
@@ -0,0 +1,48 @@
+function _getRulesFromStorage() {
+  return JSON.parse(localStorage.getItem(`${ location.origin }-shredder`))
+}
+
+function _checkParamsEquality(requestBody = {}, responseParams) {
+  const requestParams = JSON.parse(requestBody);
```

Comment:
are we always guaranteed that request body will be json? If the response contains something else it would fail with an error